sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentType.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Specialized types of Sentiment Analysis, like for example Opinion Mining.
```

Comment:
Consider updating comments when you've finalized naming decisions - specifically to clear up what's standard and what's additional.